src/cli/global/list.rs

```diff
@@ -394,6 +394,8 @@ fn format_exposed(env_name: &str, exposed: &IndexSet<Mapping>, last: bool) -> Op
     }
 }
 
+/// If the exposed name is the same as the executable name, only return the executable name
+/// otherwise return what is the exposed name and what is the executable name.
```

Comment:
```suggestion
/// If the exposed name is the same as the executable name, only return the executable name
/// otherwise return "{exposed name} -> {executable name}".
```